include/clad/Differentiator/DerivativeBuilder.h

```diff
@@ -57,13 +56,16 @@ namespace clad {
     friend class ReverseModeVisitor;
     friend class HessianModeVisitor;
     friend class JacobianModeVisitor;
-
+    
     clang::Sema& m_Sema;
     plugin::CladPlugin& m_CladPlugin;
     clang::ASTContext& m_Context;
     std::unique_ptr<utils::StmtClone> m_NodeCloner;
     clang::NamespaceDecl* m_BuiltinDerivativesNSD;
-
+    clang::FunctionDecl* cloneFunction(const clang::FunctionDecl* FD, clang::DeclContext* DC, 
+                            clang::Sema& m_Sema, clang::ASTContext&  m_Context, 
+                            clang::SourceLocation& noLoc, clang::DeclarationNameInfo name,
+                            clang::QualType functionType);     
```

Comment:
```suggestion
    clang::FunctionDecl* cloneFunction(const clang::FunctionDecl* FD,
                                                               clang::DeclContext* DC, 
                                                               clang::Sema& m_Sema,
                                                               clang::ASTContext&  m_Context, 
                                                               clang::SourceLocation& noLoc,
                                                               clang::DeclarationNameInfo name,
                                                               clang::QualType functionType);
```

We tend to follow the 80 column width -- I guess that indent should follow the rule.